test/integration/utility.py

```diff
@@ -72,3 +72,16 @@ def get_execution_duration_from_global_result_json(global_result_json):
         "the execution_duration '{} doesn't end with the expected suffix 's' for seconds".format(
             duration_json_string))
   return float(duration_json_string.rstrip('s'))
+
+
+def count_log_lines_with_substring(logs, substring):
+  """Count the number of log lines containing the supplied substring.
+
+  Args:
+    logs: Nighthawk client log output.
```

Comment:
Since we aren't using pytype, we also need to indicate the object type of each argument and return value.

Args:
  logs: A string, a log output to count the log lines in.

...

Reply:
Done.